src/cli.ts

```diff
@@ -74,7 +74,14 @@ async function runSuggestLadder(argv) {
 }
 
 async function exportWmafResultToCsv(argv) {
-    const folder = argv.folder.split("/").slice("-2").join("/");
+    let folder = "";
+    if (folder.includes("/")) {
+        folder = argv.folder.split("/").slice("-2").join("/");
+    } else if (folder.includes("\\")) {
+        folder = argv.folder.split("\\").slice("-2").join("\\");
+    } else {
+        folder = argv.folder;
+    }
```

Comment:
A bit unsure what this was doing, so didn't dare to do anything but replicate the same for windows folder separators